todo.js

```diff
@@ -16,13 +16,13 @@ function removeTodo(todos, todoId) {
 
 function changeTodoTitle(todos, todoId, newTitle) {
   return todos.map((todo) =>
-    todo.id === todoId ? Object.assign(todo, { title: newTitle }) : todo
```

Comment:
Object.assign mutates first argument. We are trying to avoid any mutation of original objects